governance/publishing-deliverables.md

```diff
@@ -0,0 +1,141 @@
+  deliverable
+* Blog posts that summarize and introduce your deliverable
+* Social media outreach of your deliverable via Twitter, LinkedIn
+
+Additionally, publishing an artifact during or just before KubeCon allows us to
+share the publication announcement widely in different forums such as Cloud
+Native SecurityCon Sessions, TAG Security Maintainer session, as well as
+serendipitous interactions in the hallways, both virtual and real world.
+
+## Define Stages of Publication
+
+Each Deliverable can go through several stages from inception to publication.
+Treating this process like a project with milestones allows folks as well as the
+lead to understand if timely progress is being made or not.
+
+A tentative schedule for a paper publication process can be found
+[here](paper-process.md#tentative-schedule-milestones)
+
+## Write a Blog
+
+A blog for a TAG Security Deliverable, is like a trailer for the movie, except
+both are published on the same day!
+
+There are several good examples of blogs about artifacts published by CNCF TAG
+Security. Here's one about
+[Cloud Native Security Whitepaper v2](https://www.cncf.io/blog/2022/05/18/announcing-the-refreshed-cloud-native-security-whitepaper/)
+
+A typical blog should include:
+
+- A brief summary about the deliverable
+- Link to the deliverable
+- Quote from one or more main contributors or project lead
+- Why should the readers care about the deliverable
+
+## Common Mistakes to Avoid
+
+Every human being is prone to mistakes. So the intent of this section is to help
+avoid making the same mistakes as past project leads so that you can make newer
+mistakes and update this section with your experiences.
+
+Some common mistakes to avoid:
+
+* **Not reviewing the artifact for brevity, correctness and flow**: Project lead
+  should attempt to review the artifact a minimum of 2-3 times with a different
+  goal for each review
+* **Missing names of contributors or reviewers of the paper**: Double check if
+  the contributors or reviewers are missing in the PDF or markdown and
+  cross-check with GitHub issue assignees who worked on the tasks for the
+  deliverables.
+* **Engaging with CNCF too close to publication date**: CNCF service desk
+  probably gets a lot of requests for help close to KubeCon. Engaging them very
+  early with a service desk ticket allows you to make progress early and creates
+  higher chance of the deliverable being published on time.
+
+## Sharing Far and Wide
+
+Publishing the artifact has little benefit if it does not reach the intended
+audiences. So seek help from CNCF, Co-chairs, Tech Leads of TAG Security to
+share the artifact through their networks. Encourage contributors and reviewers
+to share the artifact to their networks as well. Make use of CNCF mailing lists
+during Request for Comment stage and after publication to share useful anecdotes
+about the deliverable followed by a
+"Call to Action".
```

Comment:
Would be good to link to the [communications folder with the call to action template](https://github.com/cncf/tag-security/tree/main/governance/communications) for mailing list use

Reply:
done